galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/common/Dialog.kt

```diff
@@ -18,7 +18,5 @@
 package org.kopi.galite.visual.ui.vaadin.common
 
 import com.vaadin.componentfactory.EnhancedDialog
-import com.vaadin.flow.component.dependency.NpmPackage
 
-@NpmPackage(value = "@vaadin-component-factory/vcf-enhanced-dialog", version = "22.0.6")
```

Comment:
In Galite, we use the add on vcf-enhanced-dialog for fields containing rich text. 
We need to test this function to make sure that these changes do not affect the current state.
You can create a string field with the parameter styled = true to check theses changes. 
(Example in galite-demo, field  informations of TaxRuleForm)